pyproject.toml

```diff
@@ -12,9 +12,11 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.9.1,<3.10"
 opentimelineio = "^0.16.0"
-pytest = "^6.2.0"
 opencolorio = "^2.3.2"
-selenium = "^4.20.0"
+selenium = "4.16.0"
```

Comment:
what's the reasoning for downgrading selenium?
this was noted somewhere in max's original repo if i remember correctly but i can't find it.
could u explain this to me?

Reply:
Newer versions of Selenium requires python dependencies which are clashing with AYON dependency package creator. This version is the last one which could server our needs. I original Max's repository was used 4.11. We have to later properly test if this will be working, but since curretnly we are not yet finalized Slate renderer, this will need to stay as it is for now.

Reply:
okay i understand. then let's keep the version and hope for the best for the slate renderer/configurator :)